source/include/core_json.h

```diff
@@ -118,9 +120,8 @@ JSONStatus_t JSON_Validate( const char * buf,
  *
  * @return #JSONSuccess if the queryKey is found and the value output;
  * #JSONNullParameter if any pointer parameters are NULL;
- * #JSONBadParameter if the queryKey is empty, or any subpart is empty, or max is 0;
- * #JSONIllegalDocument if the buffer contents are NOT valid JSON;
- * #JSONMaxDepthExceeded if object and array nesting exceeds a threshold;
+ * #JSONBadParameter if the queryKey is empty, or any subpart is empty, or max is 0,
+ * or an index is too large to convert;
```

Comment:
The "index is too large to convert" phrase is unclear to me. What is the input index value converted to?

Reply:
as int32_t